Sources/App/Controllers/Heartbeat/HeartbeatController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  HeartbeatController.swift
+//  App
+//
+//  Created by di on 01.12.2017.
```

Comment:
Remove this comments at the top of all created files. It seems that we keep it clean in all files.

Reply:
fixed